webapp/src/js/components/panoptes/Feed.js

```diff
@@ -44,7 +49,7 @@ let Feed = createReactClass({
   },
 
   render() {
-    const {id, templateDocPath, className, actionsAreaIsClickable, actionsAreaDisappearsOnExpand, ...otherProps} = this.props;
+    let {id, templateDocPath, className, actionsAreaIsClickable, actionsAreaDisappearsOnExpand, tags, count, ...otherProps} = this.props;
```

Comment:
Why not keep these `const` and mutate a copy of `tags`?

Reply:
Fine, done.